lib/plugins/blocks.js

```diff
@@ -356,9 +356,16 @@ function inject (bot, { version, storageBuilder }) {
     for (let i = 0; i < packet.records.length; ++i) {
       const record = packet.records[i]
 
-      const blockZ = record.horizontalPos !== undefined ? (record.horizontalPos & 0x0f) : record & 0x0f
-      const blockX = record.horizontalPos !== undefined ? (record.horizontalPos >> 4) & 0x0f : (record >> 4) & 0x0f
-      const blockY = record.y !== undefined ? record.y : (record >> 8)
+      const blockX, blockY, blockZ
+      if (version.version >= 735) {
+        blockX = record.horizontalPos & 0x0f
+        blockZ = (record.horizontalPos >> 4) & 0x0f
+        blockY = record.y
+      } else {
+        blockX = record & 0x0f
```

Comment:
I'll check this properly later but in the mean time could you try this by putting console.log afterwards and doing an explosion in game and checking the position are coherent ?